File: src/main/java/com/p2p/lending/service/RechargeService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Recharge;





public interface RechargeService {
	Map<String, Object> selectrc(String currpage, Map<String, Object> findmap);
	List<Recharge> selectall();
	int sumczmoneyre();
	int sumdzmoneyre();
}

File: src/main/java/com/p2p/lending/service/RechargeService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Recharge
import org.springframework.stereotype.Service

@Service
interface RechargeService {
    fun selectrc(currpage: String, findmap: Map<String, Any>): Map<String, Any>
    fun selectall(): List<Recharge>
    fun sumczmoneyre(): Int
    fun sumdzmoneyre(): Int
}
